### ISSUE TITLE
Core 2.2.1 and In-Process Hosting

### ISSUE THREAD

**Issue by DWalkit** [arichika/RotativaCore#3]
Upon changing my .csproj to enable "In-Process Hosting" as added in Core 2.2+:
<AspNetCoreHostingModel>InProcess</AspNetCoreHostingModel>

My pdf's render without error but the CSS is ignored.  If I switch the above back to 'OutOfProcess' they render perfectly.

I know this is a brand new .NET update so I may be one of the first to run into it.  Any suggestions would be greatly appreciated.

Ahh, in my _PdfLayout.cshtml I have a call to Environment.CurrentDirectory and I recall mention of that in the Microsoft migration guide!  I had searched the entire solution for such calls and since these are in a view it missed finding them!  

I'd imagine I'll be able to fix this, so thank you and disregard!

**Comment by arichika** [comment]
Dear @DWalkit .

I upgraded the sample web application to netcoreapp 2.2 and tested it in the Azure Webn Apps environment. Even in in process mode, it worked correctly.

`netcoreapp 2.2`  version is here. (latest master)
https://github.com/arichika/RotativaCore/tree/2.2.0-updateSamples/src/RotativaCore.SampleWebApplication

I have noticed this comment since I have written so far.

> Ahh, in my _PdfLayout.cshtml I have a call to Environment.CurrentDirectory and I recall mention of that in the Microsoft migration guide! I had searched the entire solution for such calls and since these are in a view it missed finding them!

Yeah, Thats sounds good and don't worry. No problem.

For this module, I wanted to check the operation in inprocess mode. 
It's good timing. I confirmed that it works correctly in inprocess mode. 

Thank you!
